package com.Exception.Handling;

public class FinallyExample {
	
	public static void main(String [] args) {
		
		int a=10;
		int b=0;
		
		System.out.println("Value of a:" +a);
		System.out.println("Value of b:" +b);
        // System.exit(0); finally won't get execute.
        int c = a / b;
        System.out.println("a / b = " + c);
	}

}
